app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -2,26 +2,17 @@ import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
 import { Button, Pane } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
+import { Input, Textarea, Select } from "neetoui/formik";
 
-import notesApi from "apis/notes";
 import formValidationSchemas from "constants/formValidationSchemas";
 
-export default function NoteForm({ onClose, refetch, note, isEdit }) {
+import { ASSIGNED_CONTACT_OPTIONS, TAGS_OPTIONS } from "../constants";
+
+const NoteForm = ({ isEdit = false, onClose, note, handleNote }) => {
```

Comment:
What's your reasoning for naming `NoteForm` over just `Form`? I would like to understand why you are prefixing/suffixing keywords from time to time when naming components.

Reply:
This file and above mentioned files were already present in wheel and somehow I missed to check the naming conventions. Will correct all the file names in next PR.